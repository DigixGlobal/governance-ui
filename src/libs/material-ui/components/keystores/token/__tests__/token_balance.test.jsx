import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import TokenBalance from '../token_balance';

jest.mock('../../../../helpers/stringUtils', () => ({
  parseBigNumber: () => 20,
}));

const token = {
  decimals: 2,
};

const address = {
  address: '0x5434132',
};

const contract = {
  balanceOf: () => 10,
};

describe('<TokenBalance />', () => {
  test('renders correctly', () => {
    const component = shallow(
      <TokenBalance
        contract={contract}
        address={address}
        token={token}
      />,
    );

    expect(toJson(component)).toMatchSnapshot();
  });
});
