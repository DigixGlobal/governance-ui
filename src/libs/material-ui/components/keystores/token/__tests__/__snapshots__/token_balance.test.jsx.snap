// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TokenBalance /> renders correctly 1`] = `
<BalanceLabel
  item={
    Object {
      "decimals": 2,
    }
  }
  value={20}
/>
`;
